### PR TITLE
feat: added coverage

### PR DESCRIPTION
**Pull request by iharvarauka**

### CODE REVIEW

File: .github/workflows/coverage.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        required: false
+        type: string
+        default: stable
+      coverage:
+        required: false
+        type: number
+        default: 14
+
+jobs:
+  coverage:
```
Comment (maratsh) [inline .github/workflows/coverage.yaml:14]: **thought (blocking):** 
I suppose this job should be part of checks, right after test job ?

**Review by robert-affinidi — APPROVED** [review]
LGTM

File: .github/workflows/checks.yaml [hunk 2]
```diff
@@ -9,6 +9,11 @@ on:
         required: false
         type: string
         default: ""
+      coverage:
+        required: false
+        type: number
+        default: 14
```
Comment (robert-affinidi) [inline .github/workflows/checks.yaml:15]: 14%?

Comment (iharvarauka) [reply .github/workflows/checks.yaml:15]: @robert-affinidi I have checked it's our max coverage at that moment

Comment (robert-affinidi) [reply .github/workflows/checks.yaml:15]: It is a parameter which can be overridden on the repo. Let's keep it 80 by default.

**Pull request merged**